libs/accounts/src/lib/accounts-table.spec.tsx

```diff
@@ -35,7 +35,11 @@ describe('AccountsTable', () => {
   it('should render correct columns', async () => {
     await act(async () => {
       render(
-        <AccountTable rowData={singleRowData} onClickAsset={() => null} />
+        <AccountTable
+          rowData={singleRowData}
+          onClickAsset={() => null}
+          isReadOnly={false}
+        />
```

Comment:
**chore (non-blocking)**: `isReadOnly` should probably get a test too. Non-blocking a it's such simple functionality

I've posted this twice - assume it's true for all the components that now take this prop

Reply:
yeah, I guess only false was purely tested. I am adding some extra cases now